### ISSUE TITLE
"restart" works from click event but nut from handleEvent function

### ISSUE THREAD

**Issue by marmooli** [cipchk/ngx-countdown#91]
I can call restart() from a button click event manually and it works, but if do the same from the handleEvent it doesn't work:

  @ViewChild('cd', { static: false }) private countdown: CountdownComponent;

 <countdown #cd [config]="config" (event)="handleEvent($event)"></countdown>
 <button (click)="onRestart()">restart</button>

  handleEvent(event){
    if(event.action === 'done'){
      this.countdown.restart();  // this doesn't work
    }
  }

  onRestart(){
    this.countdown.restart(); // this works!
  }

**Comment by cipchk** [comment]
A temporary solution:  

```ts
import { Component, ViewChild } from "@angular/core";
import { CountdownComponent, CountdownEvent } from "ngx-countdown";

@Component({
  selector: "my-app",
  template: `
    <countdown
      #cd
      [config]="{ leftTime: 3 }"
      (event)="handleEvent($event)"
    ></countdown>
  `
})
export class AppComponent {
  @ViewChild("cd", { static: false }) private countdown: CountdownComponent;

  handleEvent(event: CountdownEvent) {
    if (event.action === "done") {
      setTimeout(() => {
        this.countdown.restart(); // this work
      })
    }
  }
}
```

**Comment by cipchk** [comment]
fixed by `11.0.2`
